lib/bundle/cask_installer.rb

```diff
@@ -1,6 +1,8 @@
 module Bundle
   class CaskInstaller
-    def self.install(name)
+    PERMITTED_OPTIONS = %w{caskroom appdir prefpanedir qlplugindir fontdir binarydir input_methoddir screen_saverdir}
```

Comment:
Cask options aren't the same as args passed to the build script of the installing brew...
Why would we treat cask the same?

Sent from my iPhone

> On Jul 18, 2015, at 11:03 PM, Mike McQuaid <notifications@github.com> wrote:
> 
> In lib/bundle/cask_installer.rb:
> 
> > @@ -1,6 +1,8 @@
> >  module Bundle
> >    class CaskInstaller
> > -    def self.install(name)
> > +    PERMITTED_OPTIONS = %w{caskroom appdir prefpanedir qlplugindir fontdir binarydir input_methoddir screen_saverdir}
> As @xu-cheng said: please remove these.
> 
> —
> Reply to this email directly or view it on GitHub.
>